tests/functional/cli/test_black.py

```diff
@@ -11,10 +11,10 @@
 from jupytext.compare import compare, compare_cells, compare_notebooks
 from jupytext.header import _DEFAULT_NOTEBOOK_METADATA
 
-from .utils import list_notebooks, requires_autopep8, requires_black, requires_flake8
+from ...utils import list_notebooks
 
 
-@requires_black
+@pytest.mark.requires_black
```

Comment:
For these tests, it would be good to refactor to:
```
try:
    import black
except ImportError:
    pytest.skip("black not available", allow_module_level=True)
```

While the other additional dependencies could be handled with `pytest.importorskip` or with `pytest_mark` (re-written explicitly). I think the modern approach nowadays is the former.